api/src/main/java/org/pmiops/workbench/billing/FreeTierBillingService.java

```diff
@@ -419,7 +419,7 @@ private Map<Long, Double> getFreeTierWorkspaceCostsFromBQ(List<WorkspaceCostView
    *
    * <ol>
    *   <li>The workspace is active
-   *   <li>The workspace is deleted and either of the following is true
+   *   <li>The workspace is deleted within the past 6 months and either of the following is true
    *       <ol>
    *         <li>Free Tier Usage last updated time is null. This means that it wasn't calculated
    *             before
```

Comment:
for the 2 cases where we are comparing the workspace's last-modified time, could you add some clarifying text that this is because what we care about it is the deletion time?

Reply:
yes I'll clarify this a bit